lasagne/layers/conv.py

```diff
@@ -117,23 +109,22 @@ class Conv1DLayer(Layer):
         An integer or a 1-element tuple specifying the stride of the
         convolution operation.
 
-    border_mode : str, one of 'valid', 'full', 'same'
-        A string indicating the convolution border mode.
+    pad : {0, (0,), 'full', 'same'} (default: 0)
```

Comment:
Ok, I guess it's `0` because `nnet.conv2d` doesn't support nonzero padding yet? This seems pretty confusing to me.

I think it should either just take `['valid', 'full', 'same']`, or we should enable padding by wrapping the input in a `PadLayer`.